packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -50,10 +85,13 @@ class UserSessionManager implements IUserSessionManager {
 
     if (!persistedAnonymousId) {
       // TODO: implement the storage.getAnonymousId autoCapture functionality as plugin that takes options in
-      const autoCapturedAnonymousId = [undefined];
+      const autoCapturedAnonymousId = defaultPluginManager.invoke<string | undefined>(
```

Comment:
On Ln 72, why invoke the plugin if we don't know for sure that it'll be used?

Reply:
done